doc/src/manual/environment-variables.md

```diff
@@ -68,23 +68,13 @@ and a global configuration search path of
 ### `JULIA_LOAD_PATH`
 
 A separated list of absolute paths that are to be appended to the variable
-[`LOAD_PATH`](@ref). (In Unix-like systems, the path separator is `:`; in Windows
```

Comment:
This should probably now say "used to set LOAD_PATH" instead of "appended to LOAD_PATH".